rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -1124,6 +1124,41 @@ void save_saveNewFederationBtcUTXOs_utxosShouldBeSavedToStorage(FederationStorag
 
     }
 
+    @Test
+    void save_saveOldFederationBtcUTXOs_utxosShouldBeSavedToStorage() {
+
+        StorageAccessor storageAccessor = new InMemoryStorage();
+        FederationStorageProvider federationStorageProvider = new FederationStorageProviderImpl(storageAccessor);
+
+        Address btcAddress = BitcoinTestUtils.createP2PKHAddress(networkParameters, "test");
+
+        DataWord oldFederationBtcUtxosKey = OLD_FEDERATION_BTC_UTXOS_KEY.getKey();
+
+        // Save utxos directly in storage.
+        List<UTXO> expectedUtxos = BitcoinTestUtils.createUTXOs(1, btcAddress);
+        storageAccessor.saveToRepository(oldFederationBtcUtxosKey, expectedUtxos, BridgeSerializationUtils::serializeUTXOList);
+
+        // Get utxos from method and check they are as expected
+        List<UTXO> actualUtxos = federationStorageProvider.getOldFederationBtcUTXOs();
+        assertEquals(1, actualUtxos.size());
+        assertEquals(expectedUtxos, actualUtxos);
```

Comment:
i think this whole part is covered by [getOldFederationBtcUTXOs test](https://github.com/rsksmart/rskj/blob/5ab6c368fa83190328789bcdc932272bbf12e55c/rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java#L694). Maybe you could just do the other part, that is actually testing the `.save()`, something like:

```
List<UTXO> utxos = federationStorageProvider.getOldFederationBtcUTXOs();
List<UTXO> extraUtxos = BitcoinTestUtils.createUTXOs(1, btcAddress);

actualUtxos.addAll(extraUtxos);
federationStorageProvider.save(networkParameters, activations);

// Getting the utxos from storage to ensure they were stored.
List<UTXO> utxosFromRepository = storageAccessor.getFromRepository(oldFederationBtcUtxosKey, BridgeSerializationUtils::deserializeUTXOList);

assertEquals(actualUtxos.size(), utxosFromRepository.size());
```

wdyt?

Reply:
We are creating tests for individual methods. `getOldFederationBtcUTXOs` has its tests, `setOldFederationBtcUTXOs` has its test and `save` should have its own test, since in real life, they are called separately.

Reply:
I agree! but you dont need the first part, you just need to ensure that calling `.save()` after adding utxos actually saves them in the storage

Reply:
Updated.